dataeng/resources/warehouse-transforms-ci.sh

```diff
@@ -53,11 +48,11 @@ then
     
     DBT_PROJECT_PATH='reporting'
     # This is a Slim CI syntax used to "run" only modified and downstream models
-    DBT_RUN_OPTIONS="-m state:modified+ @state:modified,1+test_type:data --defer --state $WORKSPACE/manifest" 
+    DBT_RUN_OPTIONS="-m state:modified+ @state:modified --defer --state $WORKSPACE/manifest"
     DBT_RUN_EXCLUDE='' ## TODO Add excluded models here
     # Will add --defer here when DBT version is upgraded
     # This is a Slim CI syntax used to "test" only modified and downstream models
-    DBT_TEST_OPTIONS="-m state:modified+ --state $WORKSPACE/manifest"
+    DBT_TEST_OPTIONS="-m state:modified+ --defer --state $WORKSPACE/manifest"
```

Comment:
dbt v0.19.0 allows using --defer state in the dbt test command

Reply:
Documentation reference: https://next.docs.getdbt.com/reference/node-selection/state-comparison-caveats/#tests